src/container/container.ts

```diff
@@ -593,15 +592,15 @@ class Container implements interfaces.Container {
     private _getNotAllArgs<T>(
         serviceIdentifier: interfaces.ServiceIdentifier<T>,
         isMultiInject: boolean,
-        key?: string | number | symbol,
-        value?: any,
+        tags?: interfaces.Tag[]
     ): GetArgs<T> {
         const getNotAllArgs: GetArgs<T> = {
             avoidConstraints: false,
             isMultiInject,
             serviceIdentifier,
-            key,
-            value,
+            key: tags?.[0]?.[0],
```

Comment:
This is to support the current interface of the `GetArgs`. I've marked these attributes as deprecated to support multiple tags in the request.